settings/types.go

```diff
@@ -60,3 +60,33 @@ type AirflowPools []struct {
 	PoolSlot        string `yaml:"slots"`
 	PoolDescription string `yaml:"description"`
 }
+
+// types for creating variables and connections yaml files
+
+type VariablesYAML struct {
+	VarYAMLs `mapstructure:"connections" yaml:"variables"`
+}
+
+type VarYAMLs []VarYAML
+
+type VarYAML struct {
+	Key   string `mapstructure:"key" yaml:"key"`
+	Value string `mapstructure:"value" yaml:"value"`
+}
+
+type ConnectionsYAML struct {
```

Comment:
```suggestion
type DAGRunConnections struct {
```
same as last comment